### DISCUSSION TITLE
Problem with Simu5G installation on Omnet++

### DISCUSSION THREAD

**Discussion by MSklodowski** [omnetpp/omnetpp#1238] [Q&A]
Hello,

I have a problem with installing the Simu5G library in the Omnet++ simulator. I am using Ubuntu version 20.04.6 LTS installed on dual boot with Windows 11. With Omnet++ version 6.0.1 and INET library version 4.4.2, the installation was successful (there were no errors because I performed "bulid project" and sample simulations worked correctly), I cannot install the simu5G library. I tried 2 methods:
using the "Install Simulation Models" method or by importing the tar.gz file for versions 1.2.0 or 1.2.1. In most cases, there were a lot of errors (including the Makefile file). In one case, the library in version 1.2.1 was installed correctly (after doing Ctrl + B, there were no errors). To be sure, I ran the "MultiMecHost" simulation from the NR directory, but the following error appeared mistake:

![image](https://github.com/user-attachments/assets/83e286de-fda2-439a-9aee-e2627e07d279)

What am I doing wrong? Should I install this library in a different way or is it due to Omnet/library/Ubuntu version?

**Comment by avarga** [comment]
I'd recommend using opp_env, see https://omnetpp.org/opp_env

**Comment by MSklodowski** [comment]
Well, I installed the Nix package manager according to the instructions on the website, but now I can't run the terminal :(

**Comment by MSklodowski** [comment]
Ok, there is a thing - I installed the INET library as soon as I created the new project ("Install INET Framework") and then simu5G. The "MultiMecHost" simulation worked. Only that they are libraries on the latest versions, and I need INET 4.4 and Simu5G 1.2.1 Why is it that doesn't work for the older version?
